Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_102.AdoPat.sarif

```diff
@@ -64,7 +64,7 @@
           "message": {
             "id": "Default",
             "arguments": [
-              "…wbbz5q",
```

Comment:
>wbbz5q [](http://example.com/codeflow?start=16&length=6)

Previously we showed a piece of the decoded token to users (when finding a base64 encoded ADO pat). This was confusing. We now reflect the data that is in the actual target file.